test.py

```diff
@@ -299,6 +300,9 @@ def normalize_points(points):
     # Print Gesture    
     cv2.putText(frame, text_a, (frame.shape[1] // 2 + 230, frame.shape[0] // 2 - 220), cv2.FONT_HERSHEY_SIMPLEX, 2, (0, 0, 255), 3)
 
+    if time.time() - visual_notification[1] < time_threshold * 2:
+        cv2.putText(frame, visual_notification[0], (frame.shape[1] // 2 + 250, frame.shape[0] // 2 + 250), cv2.FONT_HERSHEY_SIMPLEX, 2, (255, 0, 0), 3)
+

```

Comment:
Is this `cv2.putText(frame, visual_notification[0], (frame.shape[1] // 2 + 250, frame.shape[0] // 2 + 250), cv2.FONT_HERSHEY_SIMPLEX, 2, (255, 0, 0), 3)` different from Print Gesture?

Reply:
this is for recognized gesture. I will add annotation.

Reply:
fixed